translate/storage/properties.py

```diff
@@ -709,11 +733,12 @@ class stringsfile(propfile):
     Extensions = ['strings']
 
     def __init__(self, *args, **kwargs):
-        kwargs['personality'] = "strings"
+        if not kwargs.get('personality'):
+            kwargs['personality'] = "strings"
```

Comment:
This change is probably not needed now when  the code has been moved to the dialect...

Reply:
Yep! this is a leftover from my previous approach. Will remove and squash.